src/main/java/frc/robot/Constants.java

```diff
@@ -29,6 +29,9 @@
   /** Configuration options, PID constants, and CAN Bus ID's for various subsystems */
   public final static class Subsystem {
     /** Configuration options, PID constants, and CAN Bus ID's for {@link frc.robot.subsystems.Drivetrain Drivetrain} subsystem */
+    public final static class Intake {
+      public final static int TAKE_ID = 22;
```

Comment:
It's a good idea to have a boolean for whether or not the motor is inverted as well